services/catalog/tests/unit/conftest.py

```diff
@@ -215,32 +216,34 @@ def service_caching_disabled(monkeypatch: pytest.MonkeyPatch) -> None:
 
 
 @pytest.fixture
-def postgres_setup_disabled(monkeypatch: pytest.MonkeyPatch) -> None:
-    monkeypatch.setenv("CATALOG_POSTGRES", "null")
+def postgres_setup_disabled(mocker: MockerFixture):
+    mocker.patch.object(simcore_service_catalog.core.events, "postgres_lifespan")
```

Comment:
Q: is postgres not anymore nullable? I find that good, but just to understand the change here.

Reply:
Yes, I don't believe Postgres can be disabled using environment variables. I also can't think of any scenario where the catalog service would function without Postgres.